pkg/output/output_test.go

```diff
@@ -45,13 +48,27 @@ func (s *OutputSuite) TestValidateKey(c *C) {
 	}
 }
 
+// FIXME: replace this with TestLogAndParse
```

Comment:
Do you want to fix this in this PR ?

Reply:
I think this is addressed in `stream_test.go`